tests/jsmarionette/client/sockit-to-me/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "build/Release/sockit",
 
   "devDependencies": {
-    "node-gyp": "~0.10.6"
+    "node-gyp": "3.0.1"
```

Comment:
Also don't see any value in version 4.0.0 here. engines should be 0.12.* and >=4.0 it works as advertised on 0.12.